mobile-app/lib/models/learn/curriculum_model.dart

```diff
@@ -59,7 +59,7 @@ class SuperBlock {
     if (chapterBasedSuperBlocks.contains(dashedName)) {
       return SuperBlock(
         dashedName: dashedName,
-        name: name,
```

Comment:
Since this variable is replaced, we can remove it from the function arguments as it's not used anywhere else.